javascript/implementation/GinkInstance.ts

```diff
@@ -102,6 +103,18 @@ export class GinkInstance {
         this.listeners.set("all", []);
         this.initilized = true;
         //this.logger(`GinkInstance.ready`);
+        if (this.store instanceof LogBackedStore) {
```

Comment:
I'd rather not change the behavior of the GinkInstance based on the store type; that kind of breaks the point of polymorphism.  Instead I'd recommend changing the store interface to allow you to add a callback.

Also, `this.initialized = true` should always be at the end of the initialization function.

Reply:
Okay looks good thank you.